src/unit_test/CalculatorSnapshots.test.js

```diff
@@ -0,0 +1,10 @@
+import renderer from 'react-test-renderer';
+import React from 'react';
+import Calculator from '../components/Calculator';
+
+it('Calculator renders correctly', () => {
+  const tree = renderer
+    .create(<Calculator />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Using the react testing library you are required to write tests that simulate user interactions like clicking buttons and performing calculations. Kindly add such tests.

  You can visit this basic tutorial for clues on how to do that: [https://www.robinwieruch.de/react-testing-library/](https://www.robinwieruch.de/react-testing-library/)

- [ ] One of your tests is failing. Kindly fix that. I think this is coming from the `App.test.js` file.

  ![ieoe](https://user-images.githubusercontent.com/54848363/200854768-ae2fc21f-d84c-46e2-b82a-a157ea6172a1.jpg)

Reply:
Thank you! Issues resolved